carbonate/fill.py

```diff
@@ -85,8 +85,13 @@ def fill(src, dst, tstart, tstop):
             return
 
 
-def fill_archives(src, dst, startFrom):
-    header = info(dst)
+def fill_archives(src, dst, startFrom, lock):
```

Comment:
Maybe `def fill_archives(src, dst, startFrom, lock=False)` is better? No need to fix tests then.